arwen/host/asyncCall.go

```diff
@@ -259,6 +259,27 @@ func (host *vmHost) createDestinationContractCallInput(asyncCallInfo arwen.Async
 	return contractCallInput, nil
 }
 
+func (host *vmHost) computeCallValueFromVMOutput(destinationVMOutput *vmcommon.VMOutput) *big.Int {
+	if !host.IsArwenV3Enabled() {
+		return big.NewInt(0)
+	}
+
+	returnTransfer := big.NewInt(0)
+	callBackReceiver := host.Runtime().GetSCAddress()
+	outAcc, ok := destinationVMOutput.OutputAccounts[string(callBackReceiver)]
+	if !ok {
+		return returnTransfer
+	}
+
+	for _, outTransfer := range outAcc.OutputTransfers {
+		if outTransfer.Value != nil {
+			returnTransfer.Add(returnTransfer, outTransfer.Value)
```

Comment:
```suggestion
			returnTransfer = returnTransfer.Add(returnTransfer, outTransfer.Value)
```

Reply:
would leave like this. this is the way we use all over the protocol.